snips_nlu/utils.py

```diff
@@ -309,6 +309,17 @@ def wrapped(*args, **kwargs):
     return get_wrapper
 
 
+def check_persisted_path(func):
+    def func_wrapper(self, *args, **kwargs):
```

Comment:
Maybe you could have enforced `def func_wrapper(self, path)` or  `def func_wrapper(self, path, *args, **kargs)` to get an `ArgumentError` instead of an `IndexError` in case the wrapped function has no argument